src/stories/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+../../CHANGELOG.md
```

Comment:
Does Markdown support linking like this? This is going to be not-useful when browsing the codebase, right?

Reply:
So, yes it does. This has to do with the storybook present change that complains about importing files outside of the `src` directory. Not the most elegant solution, but definitely a conversation worth having to avoid symlinking

Reply:
Oh, it's a symlink!
In that case, this _might_ screw up windows users. We should probably check.

Reply:
Hmm, good point. Maybe for now I'll just drop CHANGELOG from Storybook to keep it safe